README.md

```diff
@@ -1,15 +1,27 @@
 # [fish-jabba][]
 
-[fish-jabba]:        https://github.com/2m/fish-jabba
+[fish-jabba]: https://github.com/2m/fish-jabba
 
-This is a [fish](https://fishshell.com) plugin for [jabba](https://github.com/shyiko/jabba). Together with packaging the [original shell integration code](https://github.com/shyiko/jabba/blob/1ca1e8a12f9e4c45530e2c652d79483678eef9c0/install.sh#L163-L172) required for jabba to work correctly, this plugin also includes autocompletion support for the following commands:
+This is a [fish](https://fishshell.com) plugin for [jabba](https://github.com/shyiko/jabba). T
```

Comment:
That's a bit of a strange split. :)

Reply:
Ah yes, I totally missed that.  Correcting.